.github/workflows/main.yml

```
@@ -94,7 +95,7 @@ jobs:
           path: C:/_/artifacts/*.pkg.tar.*
 
       - name: "Clean up runner"
-        if: ${{ always() }}
+        if: ${{ matrix.msystem == 'CLANGARM64' }}
```

I think this still needs always() condition in there so it runs on failed or canceled too.  Maybe
```suggestion
        if: ${{ always() && matrix.msystem == 'CLANGARM64' }}
```